tests/integration/minimal/cloudformation.json

```diff
@@ -538,7 +538,27 @@
             },
             {
               "Action": [
-                "route53:*"
+                "route53:ChangeResourceRecordSets",
+                "route53:ListResourceRecordSets",
+                "route53:GetHostedZone"
+              ],
+              "Effect": "Allow",
+              "Resource": [
+                "arn:aws:route53:::hostedzone/Z1AFAKE1ZON3YO"
```

Comment:
I think this actually has to go, this is probably something with my local environment getting in the way